src/physics/physics.cpp

```diff
@@ -48,6 +52,27 @@ PhysicsBase<dim,nstate,real>::PhysicsBase(
         manufactured_solution_function_input)
 { }
 
+template <int dim, int nstate, typename real>
+std::array<dealii::Tensor<1,dim,real>,nstate> PhysicsBase<dim,nstate,real>::convective_numerical_split_flux (
+    const std::array<real,nstate> &/*conservative_soln1*/,
+    const std::array<real,nstate> &/*conservative_soln2*/) const
+{
+    pcout << "ERROR: convective_numerical_split_flux() has not yet been implemented. Aborting..." <<std::flush;
```

Comment:
Hmm, if it is implemented for some PDEs, maybe something like "ERROR: convective_numerical_split_flux() has not yet been implemented for and overridden by the selected PDE. Aborting..." will be good.

Reply:
fixed